docs/recipes/analytics.md

```diff
@@ -2,26 +2,25 @@
 layout: guide
 ---
 
-When using a chatbot, you may be interested in analytics on its usage. This can be done via [@botpress/analytics](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-analytics) package.
+The [Botpress Analytics (@botpress/analytics)](https://github.com/botpress/botpress/tree/master/packages/functionals/botpress-analytics) package allows you to monitor the bots total user count, retention and lots more.
 
-To perform the basic setup, you need to install that package and initialize it by running something like this:
+To perform the basic setup, you will need to install the package (`npm install --save @botpress/analytics`) and initialize it by adding the following snippet to your src/index.js file
```

Comment:
path to file could be highlighted via " ` "

Reply:
Done :)